pkg/sql/opt/indexrec/index_recommendation_set.go

```diff
@@ -122,15 +186,98 @@ func (irs *IndexRecommendationSet) String() string {
 				if indexCol.Descending {
 					indexColSb.WriteString(" DESC")
 				}
+
 				indexCols[i] = indexColSb.String()
 			}
 
+			storingClause := ""
+			if indexRec.newStoredColOrds.Len() > 0 {
+				var storingSb strings.Builder
+				for i, col := range indexRec.newStoredColOrds.Ordered() {
+					colName := t.Column(col).ColName()
+					if i > 0 {
+						storingSb.WriteString(", ")
+					}
+					storingSb.WriteString(colName.String())
+				}
+				storingClause = " STORING (" + storingSb.String() + ")"
```

Comment:
This is fine for now, but in the future it might be nice to rely on existing formatting code for indexes rather than duplicating it. We already have formatting code for the create index AST nodes and for index descriptors. I wonder if it makes sense to build the recommended indexes into ASTs and use AST formatting code? The advantage would be that it'd be more robust. We have tests that make sure formatting is round-trippable, ensuring that we don't display an CREATE INDEX statement that can't be parsed.